src/lib/y2network/virtual_interface.rb

```diff
@@ -31,5 +31,14 @@ class VirtualInterface < Interface
     def self.from_connection(name, conn)
       new(name, type: conn.type)
     end
+
+    # Determines whether the interface can be renamed
+    #
+    # A virtual interface can always be renamed.
+    #
+    # @return [Boolean]
+    def can_be_renamed?
```

Comment:
is it used?

Reply:
It was :) After last night refactoring, it is not needed anymore.